### ISSUE TITLE
'indent' rule confused by extra parentheses

### ISSUE THREAD

**Issue by edg2s** [eslint/eslint#8924]
**Tell us about your environment**

* **ESLint Version:** whatever the web demo uses: http://eslint.org/demo/
* **Node Version:** see above
* **npm Version:** see above

**What parser (default, Babel-ESLint, etc.) are you using?**
see above
**Please show your full configuration:**
http://eslint.org/demo/ with 'indent' checked

**What did you do? Please include the actual source code causing the issue.**

```js
var $template = (
    $( 'div' )
        .foo()
        .bar()
);
```

**What did you expect to happen?**
'indent' rule should not be triggered, and indeed without the extra parentheses it isn't:
```js
var $template =
    $( 'div' )
        .foo()
        .bar();
```
**What actually happened? Please include the actual, raw output from ESLint.**
Complained about indentation of .foo and .bar and de-indent one step:
```
3:1 - Expected indentation of 4 spaces but found 8. (indent)
4:1 - Expected indentation of 4 spaces but found 8. (indent)
```

```js
// Fixed Code
var $template = (
    $( 'div' )
    .foo()
    .bar()
);
```

**Comment by not-an-aardvark** [comment]
Thanks for reporting, I can reproduce this issue.

Simplified example:

```js
/* eslint indent:error*/
(
    foo
        .bar
)
```
